nil/internal/collate/syncer.go

```diff
@@ -406,9 +344,11 @@ func (s *Syncer) GenerateZerostate(ctx context.Context) error {
 		return nil
 	}
 
-	if len(s.config.BlockGeneratorParams.MainKeysOutPath) != 0 && s.config.ShardId == types.BaseShardId {
-		if err := execution.DumpMainKeys(s.config.BlockGeneratorParams.MainKeysOutPath); err != nil {
-			return err
+	if len(s.config.BlockGeneratorParams.MainKeysPath) != 0 && s.config.ShardId == types.BaseShardId {
```

Comment:
Do you plan to update `gen-config` to use common generated zerostate?

https://github.com/NilFoundation/nil/blob/745ae21e083b8e84aae7f7aa8f4a8c349209a2f8/nil/cmd/nild/devnet.go#L126-L130

Reply:
Yes, but I don't really want to make it in this PR. Currently - I share file with Private+Public key between all validators. The next step will be to send only public key (it's enough for zerostate) and change gen-config with this approach.

Reply:
Ok. Let's do that in the next patch